include/git2/notes.h

```diff
@@ -153,6 +153,33 @@ GIT_EXTERN(int) git_note_create(
 	const char *note,
 	int force);
 
+/**
+ * Add a note for an object from a commit
```

Comment:
It would be insightful if this would note that no actual notes reference is written.